XamlControlsGallery/SettingsPage.xaml

```diff
@@ -55,12 +56,25 @@
                     <CheckBox x:Name="spatialSoundBox" Unchecked="spatialSoundBox_Unchecked" Checked="spatialSoundBox_Checked" IsEnabled="False" Content="Spatial Audio"/>
                     <TextBlock Text="See the Sound page for how to enable sound in your app!" Style="{ThemeResource CaptionTextBlockStyle}" FontStyle="Italic" Margin="0,8,0,0"/>
                 </StackPanel>
+                
                 <TextBlock Style="{StaticResource TitleTextBlockStyle}" Margin="0,40,0,0" FontWeight="Normal" Text="Navigation Style" />
                 <muxc:RadioButtons x:Name="navigationLocation" Margin="0,10,0,0"
                         Header="Navigation position" SelectionChanged="navigationLocation_SelectionChanged">
                     <RadioButton Content="Left"/>
                     <RadioButton Content="Top"/>
                 </muxc:RadioButtons>
+
+                <TextBlock
+                    Style="{StaticResource TitleTextBlockStyle}"
+                    Margin="0,40,0,0"
+                    FontWeight="Normal"
+                    Text="Screenshots" />
+                <StackPanel Margin="0,10,0,0">
+                    <ToggleSwitch x:Name="screenshotModeToggle" Header="Screenshot mode" Toggled="screenshotModeToggle_Toggled" OffContent="Off" OnContent="On" Margin="0,0,0,4"/>
+                    <Button Content="Set screenshot folder" Click="FolderButton_Click"/>
+                    <TextBlock x:Name="screenshotFolderTextBlock" IsTextSelectionEnabled="True"/>
```

Comment:
I think it would be awesome if you could also open the folder from inside the XAML Controls Gallery by clicking on a button, which would open the folder in explorer.

Reply:
Good idea! I changed it to a link.